src/Macros/BlueprintMacros.php

```diff
@@ -24,5 +24,25 @@ public static function boot(): void
             /** @var class-string<Model> $model */
             return $this->foreignSnowflake($column ?? (new $model())->getForeignKey());
         });
+
+        Blueprint::macro('morphsSnowflake', function (string $name, ?string $indexName = null): void {
```

Comment:
To keep things consistent with the framework, please rename to:

```suggestion
        Blueprint::macro('snowflakeMorphs', function (string $name, ?string $indexName = null): void {
```

same for the other one on all of these